index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div id="wrapper">
+    <section id="section-1">
+        <nav id="nav-bar">
+            <p>Logo Placeholder</p>
```

Comment:
I think it is better to wrap the logo with an `<a>` tag

[optional] you should personalize the logo with your name or personal brand 😄